packages/module/patternfly-docs/content/extensions/virtual-assistant/examples/demos/AttachmentDemos.md

```diff
@@ -46,12 +46,50 @@ import PFIconLogoReverse from '../ChatbotHeader/PF-IconLogo-Reverse.svg';
 
 ### Attach via upload button in message bar
 
+The unique features demonstrated in this demo include:
+- A `<ChatbotContent>` and [`<MessageBox>`](/patternfly-ai/chatbot/chatbot#chatbot-content-and-message-box) components are housing:
+  1. an initial user [`<Message>`](/patternfly-ai/chatbot/chatbot-messages) with an attachment included in the message
+  2. the ability to preview or edit the attachment using the `<PreviewAttachment>` and `<AttachmentEdit>` components
+  3. a PatternFly [`<Alert>`](/components/alert) to provide success or error messages for attachments.
+- The `<ChatbotContent>` and `<ChatbotFooter>` wrapped in a a `<FileDropZone>` component to enable attachments to be dragged into the chatbot.
+  1. displaying attached files in the footer using a [`<FileDetailsLabel>`](patternfly-ai/chatbot/chatbot-attachments#interactive-chip-representing-uploading-file) component
+  2. how to define custom handling of uploading a file using `<MessageBar>`'s `handleAttach` prop and `<FileDropZone>`'s `handleFileDrop` prop.
+
+In addition to the unique features, this demo also includes:
+- The [`<ChatbotToggle>`](/patternfly-ai/chatbot/chatbot-toggle) being able to toggle to the [`<Chatbot>`](/patternfly-ai/chatbot/chatbot) container.
+- A [`<ChatbotHeader>`](/patternfly-ai/chatbot/chatbot-header) with some subcomponents laid out, including a `<ChatbotHeaderTitle>` which changes its presentation depending on the display mode.
+- The ability to swap display modes via the `<ChatbotHeaderOptionsDropdown>`
+- A `<ChatbotContent>` and [`<MessageBox>`](/patternfly-ai/chatbot/chatbot#chatbot-content-and-message-box) components are housing:
+  1. an implemented `<ChatbotWelcomePrompt>`
+  2. the ability to preview or edit the attachment using the `<PreviewAttachment>` and `<AttachmentEdit>` components
+  3. an initial bot message
+  4. logic for enabling auto-scrolling to the most recent message whenever a new message is sent or received using a `scrollToBottomRef`
+- A [`<ChatbotFooter>`](/patternfly-ai/chatbot/chatbot-footer) housing a [`<ChatbotFootNote>`](/patternfly-ai/chatbot/chatbot-footer#footnote-with-popover) and a `<MessageBar>` housing the implementation of:
+
```

Comment:
```suggestion
This demo displays unique attachment features, including:
1. [`<ChatbotContent>` and `<MessageBox>`](/patternfly-ai/chatbot/chatbot#chatbot-content-and-message-box) components that contain:
  - An initial [user `<Message>`](/patternfly-ai/chatbot/chatbot-messages/#user-messages) with an attachment.
  - The ability to preview or edit the attachment using the [`<PreviewAttachment>` and `<AttachmentEdit>` components.](patternfly-ai/chatbot/chatbot-messages/#messages-with-attachments)
  - A PatternFly [`<Alert>`](/components/alert) to provide success or error messages for attachments.
2. `<ChatbotContent>` and `<ChatbotFooter>`, wrapped in a `<FileDropZone>` component to support drag and drop attachments. The footer also:
  - Displays attached files, using a [`<FileDetailsLabel>`](patternfly-ai/chatbot/chatbot-attachments#interactive-chip-representing-uploading-file) component.
  - Demonstrates custom handling of file uploads, using the `handleAttach` in `<MessageBar>` and `handleFileDrop` in `<FileDropZone>`.

This demo includes broader chatbot features, including:
1. A [`<ChatbotToggle>`](/patternfly-ai/chatbot/chatbot-toggle) that controls the [`<Chatbot>`](/patternfly-ai/chatbot/chatbot) container.
2. A [`<ChatbotHeader>`](/patternfly-ai/chatbot/chatbot-header) that updates based on the display mode, with sub-components (including a `<ChatbotHeaderTitle>`).
3. The ability to swap display modes via the `<ChatbotHeaderOptionsDropdown>`
4. A `<ChatbotContent>` and [`<MessageBox>`](/patternfly-ai/chatbot/chatbot#chatbot-content-and-message-box) with:
  - A `<ChatbotWelcomePrompt>`
  - The ability to preview or edit the attachment.
  - An initial bot message
  - Logic for enabling auto-scrolling to the most recent message, whenever a new message is sent or received using `scrollToBottomRef`
5. A [`<ChatbotFooter>`](/patternfly-ai/chatbot/chatbot-footer) with a [`<ChatbotFootNote>`](/patternfly-ai/chatbot/chatbot-footer#footnote-with-popover) and a `<MessageBar>`

```

Does the alert component actually get used? I maybe just couldn't figure out how to trigger it in the demo, but if it isn't we should remove mention of it

Reply:
I just double checked! If I tried to upload a file that was way too big, it triggered an error alert.